### ISSUE TITLE
ProGe: generate a default version of the tta0_imem_mau_pkg.vhdl

### ISSUE THREAD

**Issue by pjaaskel** [cpc/tce#114]
It's a bit illogical now that ProGe cannot generate a functioning processor without generating a program for it?

**Comment by Joonari** [comment]
You don't need an actual program for this, but you do need to run PIG to generate the required imem_mau_pkg.vhdl. PIG just needs the ProGe-generated RTL directory as an argument.

**Comment by pjaaskel** [comment]
Yes, this is confusing to users, it would be best to have the imem fetch width in IDF.
